### PR TITLE
Preloader UI updates

### LINKED ISSUE #806: PreLoader UI Update
[Reference](https://codepen.io/webcraftsman/pen/xxgvzwj)

- Change the background of the current pre-loader to the shade of the black shown above
- The atom animation should remain the same
- Add the Fade in Text Animation and change the text to a quote like "I could either watch it happen or be a part of it". The prepositions should not be made bold
- Make sure the preloader works for all the pages and all screen sizes (Use chrome dev tools).
- In the PR description, upload a clip of the the above testing

### PR DESCRIPTION
**Pull request by rachejazz**
Signed-off-by: rachejazz <divyadeepti2000@gmail.com>

# Fixes #806 
Attached gif:
demo

### CODE REVIEW

File: travel_temp.html [hunk 1]
```diff
@@ -128,4 +131,4 @@ <h2>Trip to New Delhi</h2>
         <script src="assets/js/travel_temp.js"></script>
     </body>
 
-</html>
\ No newline at end of file
+</html>
```
Comment (smaranjitghose) [inline travel_temp.html:134]: Do not make these changes as it would cause merge conflict for others!

File: projects.html [hunk 2]
```diff
@@ -61,8 +62,11 @@
             <div class="electron"></div>
             <div class="electron-alpha"></div>
             <div class="electron-omega"></div>
-        </div>
-    </div>
+		</div>
+		<div>
+			<p id="animate" style="opacity: 0;" class="text-anime">I could either watch it happen or be a part of it.</p>
```
Comment (smaranjitghose) [inline projects.html:67]: This does not make sense at all and invalidates the entire point of having an animation!

Please remove the line

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
> Signed-off-by: rachejazz [divyadeepti2000@gmail.com](mailto:divyadeepti2000@gmail.com)
> 
> # Fixes #806
> Attached gif:
> demo

Why is the nav bar messed up in your clip?

File: assets/js/preloader.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+window.onload = fadeIn;
+function fadeIn() {
+    var fade = document.getElementById("animate");
+    var opacity = 0;
+    var intervalID = setInterval(function() {
+
+        if (opacity < 1) {
```
Comment (smaranjitghose) [inline assets/js/preloader.js:7]: Do not create a separate script for this. Use the app.js

File: projects.html (hunk 2 above)
Comment (rachejazz) [reply projects.html:67]: This line adds the quote you wanted in the fadein animation. Which line do you want removed?

File: travel_temp.html (hunk 1 above)
Comment (rachejazz) [reply travel_temp.html:134]: That had a bad character ending :') but I'll revert that surely

File: projects.html (hunk 2 above)
Comment (rachejazz) [reply projects.html:67]: And the id and class tags are later used for the script to run. How will the animation occur then?

Comment (smaranjitghose) [reply projects.html:67]: Having the quote beside the animation makes the UI look worse. Put it either above or below the atom animation

Comment (rachejazz) [reply projects.html:67]: Aahh gotcha. Yeah I'll do that

**Comment by rachejazz** [comment]
Please check the current update:
demo

**Comment by rachejazz** [comment]
@smaranjitghose added the changes, do check

**Comment by smaranjitghose** [comment]
Pre_Loader


- Remove the John Doe Text and place the quote there
- Change the color of the axes in the illustrations please (since the background is already a shade of black, the jet black axes won't go well)

**Comment by rachejazz** [comment]
This is neater maybe, do give your suggestions
image

**Comment by rachejazz** [comment]
demo

**Review by smaranjitghose — APPROVED** [review]

**Pull request merged**